mgnify-pipelines-toolkit/1.0.2/env.yaml

```diff
@@ -0,0 +1,9 @@
+channels:
+- conda-forge
+- bioconda
+dependencies:
+- python =3.11
+- pandas =2.0.2
```

Comment:
I'm not sure that this is any faster or smaller than just letting pip install it as a dep of the toolkit?

Reply:
I was having issues with pandas, the wheel is not avaialbe for this combination and it was trying to compile -stuff-. So, I went with conda for it.

Reply:
Ah fair enough. I was building with `--platform linux/amd64`; could not get any combo of base images/packages to work smoothly with arm64.